src/odemis/util/driver.py

```diff
@@ -240,4 +240,3 @@ def get_backend_status():
         logging.exception("Unresponsive back-end")
         return BACKEND_DEAD
 
-    return BACKEND_DEAD
```

Comment:
That's indeed unreachable code. However, I'd rather leave it here, in case we change the code, to ensure the function always return something (proper).